source/docs/03-metadata.xml

```diff
@@ -1625,6 +1625,43 @@
                               <specDesc key="handList"/>
                            </specList>
                         </p>
+                        <p>
+                           <specList>
+                              <specDesc key="addDesc"/>
+                           </specList>
+                        </p>
+                        <p>In manuscripts there can be various kinds of entries, additions, corrections, marginalia and revisions; all these interventions in the “original” manuscript can be documented under <gi scheme="MEI">addDesc</gi>. However, it is important to understand that these are not additions to the musical text directly. These additions to the manuscript can come from the composer himself, copyists, typesetters, publishers, previous owners or librarians: In accordance with standard scholarly practice, these entries are usually indicated in the diligence text with an indication of the localisation within the manuscript as well as the means of writing used. E.g. “Auf fol. 109v links mit Bleistift von Schindler „NB (Sch.) Hier fehlen 8 Takte (auch im Chor). jedoch die eine spätere […]“” (see <ref target="https://digital.staatsbibliothek-berlin.de/werkansicht?PPN=PPN756658373&amp;PHYSID=PHYS_0398&amp;DMDID=DMDLOG_0005">facsimile</ref>).
```

Comment:
```suggestion
                        <p>In manuscripts there can be various kinds of entries, additions, corrections, marginalia and revisions; all these interventions in the “original” manuscript can be documented under <gi scheme="MEI">addDesc</gi>. However, it is important to understand that these are not additions to the musical text directly. These additions to the manuscript can come from the composer himself, copyists, typesetters, publishers, previous owners or librarians: In accordance with standard scholarly practice, these entries are usually indicated in the diligence text with an indication of the location within the manuscript as well as the means of writing used. E.g. “Auf fol. 109v links mit Bleistift von Schindler „NB (Sch.) Hier fehlen 8 Takte (auch im Chor). jedoch die eine spätere […]“” (see <ref target="https://digital.staatsbibliothek-berlin.de/werkansicht?PPN=PPN756658373&amp;PHYSID=PHYS_0398&amp;DMDID=DMDLOG_0005">facsimile</ref>).
```

Reply:
Is "diligence text" a technical term? If it isn't, then I am not sure what is meant here.

Reply:
I'm not sure if it was @doerners or @gucl-mu who introduced this text. I have the same question as @mss2221: What does `diligence tex` mean here?

Reply:
It wasn't me ;) so not sure what "diligence text" means either.

Reply:
Ok. then take out diligence. But that kind of makes "indicated ...with an indication" stand out. 

Does this work:
"These entries are usually included [or "recorded"] in the text [or maybe "in a diplomatic transcription", if that would be accurate here -- because I am not sure which 'text' is meant here] with an indication of the location within the manuscript ... "

Reply:
Fließtext not Fleißtext... so flowing text or continuous text was meant

Reply:
"These transcription are usually included in the flowing text with an indication of the location within the manuscript ... "???